src/containers/Category.js

```diff
@@ -1,8 +1,35 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
+import { getBusinessesInStateCityAndCategory } from '../helpers/db';
 
 export const Category = () => {
   const { state, city, category } = useParams();
+  const [businesses, setBusinesses] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
```

Comment:
It looks like useEffect is synchronous. Here is the eslint output from the console when I tried that.

```
Compiled with warnings.

./src/containers/Category.js
  Line 9:13:  Effect callbacks are synchronous to prevent race conditions. Put the async function inside:

useEffect(() => {
  async function fetchData() {
    // You can await here
    const response = await MyAPI.getData(someId);
    // ...
  }
  fetchData();
}, [someId]); // Or [] if effect doesn't need props or state

Learn more about data fetching with Hooks: https://fb.me/react-hooks-data-fetching  react-hooks/exhaustive-deps
```

An article with more info.
https://www.robinwieruch.de/react-hooks-fetch-data